docs/boards/Board - CrazyBeeF3FR.md

```diff
@@ -63,12 +63,15 @@ https://www.banggood.com/Racerstar-Crazybee-F3-Flight-Controller-4-IN-1-5A-1S-Bl
 
 ## Designers
 
-
-
 ## Maintainers
 
-
 ## FAQ & Known Issues
 
+- FRSKY version only: To bind to your Taranis, you need to be running the non-eu OpenTX version, which allows you to use the required D8 setting to bind to the RX. Also don't forget to set the receiver mode in BF to FRSKY_D, because the factory default is FRSKY_X.
```

Comment:
FrSky X is now supported with #6076, which will be in 3.4, so this should be changed to reflect this.

Reply:
Thinking about it, I am not sure if it makes sense to start listing what did not work with past releases, or if it will be less confusing to ask users to upgrade to 3.4 (RC1 is out now), and list the problems that still remain.
The status as it is now in 3.4 is:
- FrSky X (8 / 16 channels) and FrSky D (8 channels) work both reliably, including in combination with crash flip / Dshot beacon, as long as the TELEMETRY feature is disabled;
- basic telemetry information like RSSI and battery voltage will be sent even when the TELEMETRY feature is disabled;
- on FrSky D, the TELEMETRY feature causes occasional dropouts, depending on how many sensors (BARO, GPS, ...) are enabled, probably due to a timing overrun;
- on FrSky X, the TELEMETRY feature causes hard lockups due to a bug in the telemetry generation code.

Reply:
That's a lot of very useful information, that I did not know of. I just woke and read this, sipping my coffee, and it already made my day ;-) Thank you for that. About the listing of things-broken in past releases: Having a complete and / or consistent list of that for all older versions of BF is indeed hardly desirable or feasible. But in this case / for this specific board, it will give people an idea of what is broken on the board, by default, in the factory supplied version of BF. It will help them identify problems they might be having due to that they haven't upgraded the factory default BF. I know it would have saved me a couple of hours of work if this info had been available on the BF wiki instead of on fora and blogs, or having to experiment a long time to find out about a certain issue for myself. Also, it would have given me an idea of what would change if I upgraded the factory default version of BF.

Also, this version specific information is only in the board-specific sub-sub-items of the wiki, so it won't really clutter up any of the main documentation, if at all.

So if you agree, I would still like to put the stuff that is fixed in 3.4 in the FAQ / issue list, under the sub-header "Issues with factory supplied BF 3.3.0:" , or something like it.

Reply:
@jhalfmoon: Yes, I am fine with that. But I think the information in the pull request should be updated to reflect that FrSky X (or 'D16' as OpenTX calls it) is now supported.

Reply:
I totally agree. I'll arrange it later today.

Reply:
A bit later than promised, but... better?

Reply:
My Snapper7 board FC CrazybeeF3 BNF Frysky,
my latest firmware : betaflight_3.3.3_crazybeeF3FR (Stable)
SPI_Rx Support Receiver Mode

I accidently choosing wrong SPI Reciever Provider to A7105_Flysky_2A and save it

now my FC can not connect to USB anymore, everytime connect it is says " Unkown USB device (Device descriptor request Failed) for windows 10"

is there any suggestions to fix it? please help me anyone..